src/bermudan.cpp

```diff
@@ -162,8 +165,8 @@ Rcpp::List bermudanSwaptionEngine(Rcpp::List rparam,
     QuantLib::Rate dummyFixedRate = 0.03;
     boost::shared_ptr<QuantLib::IborIndex> indexSixMonths(new QuantLib::Euribor6M(rhTermStructure));
         
-    QuantLib::Date startDate = calendar.advance(settlementDate, 1, QuantLib::Years, floatingLegConvention);
-    QuantLib::Date maturity = calendar.advance(startDate, 5, QuantLib::Years, floatingLegConvention);
+    //QuantLib::Date startDate = calendar.advance(settlementDate, 1, QuantLib::Years, floatingLegConvention);  //took out hard coded 
+    //QuantLib::Date maturity = calendar.advance(startDate, 5, QuantLib::Years, floatingLegConvention);         //dates
```

Comment:
I can do a default with a warning. The only other issue with a default is it is specific to a calendar, so if the user is assuming a calendar, but I'll default it to US and flag that change as well.